todo_project/settings/production.py

```diff
@@ -0,0 +1,52 @@
+from .base import *  # noqa: F403
+import os
+
+# Create the logs directory if it does not exists
+os.makedirs("./logs", exist_ok=True)
```

Comment:
- I think we should add logs when we're certain of what things to log and how we want out logging solution to work in production as we have not thought about this yet, let's remove this for now, what do you think?

Reply:
Agreed. We can take this up in a separate ticket